boards/riscv/esp32c3_devkitm/Kconfig.defconfig

```diff
@@ -9,7 +9,7 @@ config BOARD
 
 config HEAP_MEM_POOL_SIZE
 	default 98304 if WIFI
-	default 16384 if BT
+	default 40960 if BT
```

Comment:
For the `esp32c3_devkitm` , when I tested it worked with `32768` also. Is the extra bit required?

Reply:
It should work with 32k. But it depends on the BLE heap usage. So I'd rather keep as is to have room for BLE stuff and leave it to the user to decrease if needed.